make/dev/npm-installer/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+cd /harbor_src/ui_ng
+rm -rf dist/*
+
+npm_proxy=
+
+while getopts p: option
+do
+    case "${option}"
+    in
+		p) npm_proxy=${OPTARG};;
+    esac
+done
+
+if [ ! -z "$npm_proxy" -a "$npm_proxy" != " " ]; then
+	npm config set proxy $npm_proxy
+fi
+
+cat ./package.json
+npm install
+./node_modules/.bin/ngc -p tsconfig-aot.json
+
+npm install
```

Comment:
@steven-zou 
The first ``npm install & ngc -p tsconfig-aot.json`` means that set up the Harbor-UI library.
The second one means that install Harbor-UI library in ``src/ui_ng`` environment. 
If I didn't execute these commands, dependency errors occurred.